official/transformer/v2/transformer_main.py

```diff
@@ -155,10 +155,9 @@ def train(self):
     # TODO(guptapriya): Figure out a way to structure input that works in both
```

Comment:
I think we can remove this TODO as we dropped the "if".  Hate to ask for this but code cleanup often gets lost and the next person will wonder if it can be removed.

Reply:
haha, thanks!  I didn't notice this.  I am happy to remove it.